scipy/stats/mstats_basic.py

```diff
@@ -2297,14 +2302,15 @@ def skew(a, axis=0, bias=True):
 
     """
     a, axis = _chk_asarray(a,axis)
-    n = a.count(axis)
```

Comment:
Only needed `if not bias and zero is not ma.masked and m2 is not ma.masked`, so moved there